Terasology/MetalRenegades
# Add basic fetch quests

**Requirements:**
 - Terasology/Minimap#11
 - Terasology/Tasks#13

**To Test:**
 - Go to a quest point in church and receive an item with the quest
 - Activate the item and complete the tasks
 - When the last task is started, the quest point that you started with will now be a beacon. Go back to the same point to complete the quest.
 - The collected items should be removed from the inventory and the player will get 50 cash.

**Issues:**
 - Because `beaconId`s are the same, if the same quest is taken multiple times, players don't need to go back to the beacon. Quests complete as soon as they collect items.
 - The beacon overlay marks the wrong point on the minimap.

**Possible Improvements:**
 - Add quests to citizens and have some dialog, allowing the player to accept/decline the quest
